connectioncmd/input.go

```diff
@@ -0,0 +1,390 @@
+		config:         config,
+		connectionType: "Local",
+		state:          0,
+		cursorMode:     textinput.CursorStatic,
+	}
+	for i := range m.inputs {
+		t := textinput.New()
+		t.CursorStyle = selectedItemStyle
+		t.SetCursorMode(textinput.CursorStatic)
+		switch i {
+		case 0:
+			t.Prompt = "- Cluster Name: "
+			t.Focus()
+			t.Placeholder = "dev"
+			t.PromptStyle = selectedItemStyle
+			t.TextStyle = selectedItemStyle
+		case 1:
+			t.Prompt = "- Member Addresses: "
+			t.PromptStyle = noStyle
+			t.Placeholder = "127.0.0.1:5701"
+			t.TextStyle = noStyle
+		case 2:
+			t.Prompt = "- Setup SSL? (T/F): "
+			t.Placeholder = "F"
+			t.PromptStyle = noStyle
+			t.TextStyle = noStyle
+		}
+		m.inputs[i] = t
+	}
+	return m
+}
+
+func RemoteInput(config *config.Config) InputModel {
+	m := InputModel{
+		inputs:         make([]textinput.Model, 3),
+		connectionType: "Remote",
+		config:         config,
+		state:          0,
+		cursorMode:     textinput.CursorStatic,
+	}
+	for i := range m.inputs {
+		t := textinput.New()
+		t.CursorStyle = selectedItemStyle
+		t.SetCursorMode(textinput.CursorStatic)
+		switch i {
+		case 0:
+			t.Prompt = "- Cluster Name: "
+			t.Focus()
+			t.PromptStyle = selectedItemStyle
+			t.TextStyle = selectedItemStyle
+		case 1:
+			t.Prompt = "- Member Addresses: "
+			t.PromptStyle = noStyle
+			t.TextStyle = noStyle
+		case 2:
+			t.Prompt = "- Setup SSL? (T/F): "
+			t.PromptStyle = noStyle
+			t.TextStyle = noStyle
+		}
+		m.inputs[i] = t
+	}
+	return m
+}
+
+func SSLInput(config *config.Config, connectionType string) InputModel {
+	m := InputModel{
+		inputs:         make([]textinput.Model, 4),
+		config:         config,
+		connectionType: connectionType,
+		state:          2,
+		cursorMode:     textinput.CursorStatic,
+	}
+	var t textinput.Model
+	for i := range m.inputs {
+		t = textinput.New()
+		t.CursorStyle = selectedItemStyle
+		switch i {
+		case 0:
+			t.Prompt = "- CA certificate path: "
+			t.Focus()
+			t.PromptStyle = selectedItemStyle
+			t.TextStyle = selectedItemStyle
+			t.SetCursorMode(textinput.CursorStatic)
+		case 1:
+			t.Prompt = "- SSL certificate path: "
+			t.PromptStyle = noStyle
+			t.TextStyle = noStyle
+			t.SetCursorMode(textinput.CursorStatic)
+		case 2:
+			t.Prompt = "- SSL key path: "
+			t.PromptStyle = noStyle
+			t.TextStyle = noStyle
+			t.SetCursorMode(textinput.CursorStatic)
+		case 3:
+			t.Prompt = "- SSL Password: "
+			t.EchoMode = textinput.EchoPassword
+			t.EchoCharacter = '•'
+			t.PromptStyle = noStyle
+			t.TextStyle = noStyle
+			t.SetCursorMode(textinput.CursorStatic)
+		}
+		m.inputs[i] = t
+	}
+	return m
+}
+
+func ApprovalInput() InputModel {
+	m := InputModel{
+		inputs:         make([]textinput.Model, 1),
+		connectionType: "Approval",
+		state:          1,
+		cursorMode:     textinput.CursorStatic,
+	}
+	t := textinput.New()
+	t.CursorStyle = selectedItemStyle
+	t.Prompt = "Your config file will be overwritten, are you want to continue? (y/n): "
```

Comment:
"do you want to continue?"

Reply:
Fixed at https://github.com/hazelcast/hazelcast-commandline-client/pull/139/commits/8a912812f9624431cb43dd10992a1e2135072552.